types/fluxible/index.d.ts

```diff
@@ -90,7 +90,7 @@ export class Fluxible {
      * @param callback
      * @async Rehydration may require more asset loading or async IO calls
      */
-    rehydrate(state: any): void;
+    rehydrate(state: any, callback: () => void): void;
```

Comment:
i think it should be callback: (err: Error, context: FluxibleContext) => void

please fix

Reply:
You are absolutely right. I even noticed the parameter was missing because I had a function with that signature. Sorry there and thanks for catching it up.

It's fixed now.

Reply:
And it should be optional, making it so.

Reply:
right, thanks for the fix, please fill free to commit new fixes if find any